minecraft_servers/theflare/manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+  "server_name": "theflare",
+  "nice_name": "theFLARE",
+  "direct_ip": "trueflare.net",
+  "supported_languages": [
+    "en"
+  ],
+  "social": {
+    "web": "https://trueflare.net",
+    "web_shop": "https://store.trueflare.net",
+    "discord": "https://discord.gg/WbQckwVyrt"
+  },
+  "gamemodes": {
+    "lifesteal": {
+      "name": "Lifesteal SMP",
+      "color": "#cf3e3e",
+      "versions": "1.20.1+"
+    }
+  },
+  "discord": {
+    "server_id": 1119215380745945098,
+    "rename_to_minecraft_name": true
```

Comment:
`rename_to_minecraft_name` = `true` is only allowed for LabyMod partners. See here: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md?plain=1#L27